numpy/_core/include/numpy/dtype_api.h

```diff
@@ -141,6 +149,7 @@ typedef struct {
 #define NPY_METH_unaligned_contiguous_loop 8
 #define NPY_METH_contiguous_indexed_loop 9
 
+
```

Comment:
`_NPY_METH_get_loop` is still marked as private. Maybe we should just make it public given it's pretty useful?

Reply:
yah, got to make it public, but we could follow up also to keep the diff smaller.

(There are a few things I dislike about it, but we aren't gonna change them quickly, and that is OK)